google-osconfig-agent.service

```diff
@@ -9,6 +9,8 @@ Restart=always
 RestartSec=1
 StartLimitInterval=120
 StartLimitBurst=3
+KillMode=mixed
```

Comment:
Why change this?

Reply:
SIGTERM was getting sent to all processes, which was causing issues with things like dpkg and left them in a messy state. If i read the docs correctly this should only send SIGTERM to osconfig, then SIGKILL to all after the timeout